python_modules/libraries/dagster-dbt/dagster_dbt/asset_utils.py

```diff
@@ -248,7 +265,10 @@ def get_dbt_multi_asset_args(
             io_manager_key=io_manager_key,
             description=default_description_fn(node_info, display_raw_sql=False),
             is_required=False,
-            metadata=default_metadata_fn(node_info),
+            metadata={
+                **default_metadata_fn(node_info),
+                MANIFEST_METADATA_KEY: manifest,
+            },
```

Comment:
Could we add this as metadata on the asset definition instead of the asset definition's outputs?

Located in [1], where we do it for `dagster-dbt/select` and `dagster-dbt/exclude`.

Reply:
I agree that would be preferable, however `op_tags` doesn't accept arbitrary Python objects.

If we went with the proposal raised [here](https://github.com/dagster-io/internal/discussions/6143), then we could add it as a field on `DbtAssetsDefinition`.